src/mmw/apps/modeling/mapshed/calcs.py

```diff
@@ -182,8 +185,10 @@ def animal_energy_units(geom):
         values = cursor.fetchone()  # Only one row since aggregate query
         aeu = dict(zip(columns, values))
```

Comment:
Consider changing `aeu` here to `population` since it's not an AEU until you multiply by weight and divide by 1000.